tests/OptionsTest.php

```diff
@@ -0,0 +1,17 @@
+<?php
+namespace ZendDeveloperToolsTest;
+
+use PHPUnit_Framework_TestCase;
+use ZendDeveloperTools\Options;
+
+class OptionsTest extends PHPUnit_Framework_TestCase
+{
+    public function testStatusOfDefaultConfiguration()
+    {
+        $dist = require __DIR__."/../config/zenddevelopertools.local.php.dist";
```

Comment:
This is only an idea but if a future change broken the default conf? We can
use a data provider to tests other stuff in other tests..

2015-03-27 11:35 GMT+01:00 Marco Pivetta <notifications@github.com>:

> In tests/OptionsTest.php
> <https://github.com/zendframework/ZendDeveloperTools/pull/179#discussion_r27286412>
> :
>
> > @@ -0,0 +1,17 @@
> > +<?php
> > +namespace ZendDeveloperToolsTest;
> > +
> > +use PHPUnit_Framework_TestCase;
> > +use ZendDeveloperTools\Options;
> > +
> > +class OptionsTest extends PHPUnit_Framework_TestCase
> > +{
> > +    public function testStatusOfDefaultConfiguration()
> > +    {
> > +        $dist = require __DIR__."/../config/zenddevelopertools.local.php.dist";
>
> Why do you need real options? Put the array directly here
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendframework/ZendDeveloperTools/pull/179/files#r27286412>
> .
>



-- 
Gianluca Arbezzano
www.gianarb.it